baselines/ippo/run_sb3_ppo.py

```diff
@@ -41,7 +41,7 @@ def train(exp_config: ExperimentConfig, scene_config: SceneConfig):
     env = SB3MultiAgentEnv(
         config=env_config,
         scene_config=scene_config,
-        max_cont_agents=env_config.MAX_CONTROLLED_VEHICLES,
+        max_cont_agents=env_config.max_num_objects,
```

Comment:
tbh I think this is kind of a confusing name. An object doesn't distinguish between static and non-static objects. Don't have an obvious solution yet